packages/nightingale-track/src/config.ts

```diff
@@ -305,8 +305,8 @@ export const config = {
     shape: "rectangle",
     color: "#8585fc",
   },
-  PTM: {
-    name: "ptm",
+  RESIDUE: {
+    name: "residue",
     label: "PTM residue",
```

Comment:
Does this need to change too?

Reply:
https://github.com/ebi-webcomponents/nightingale/blob/canvas-uniprot-fix/packages/nightingale-track-canvas/src/nightingale-track-canvas.ts#L158, I call this in two places (nightingale-track and canvas) where 'PTM' is out of context.